src/pages/Home.vue

```diff
@@ -218,8 +224,23 @@ const events = {
   onClickUpdate(item: Item) {
     itemStore.updateItem(item, props.podcastId, docname)
   },
-  onClickShare(item: Item, destination: string) {
-    shareStore.sendItem(item, destination, props.podcastId)
+  onClickShare(item: Item, destination: string = "", podcastNameToShare: [],getDeletePodcastId: string = "") {
+    if(destination !== ""){
+      shareStore.sendItem(item, destination, props.podcastId)
+    }
+    if(getDeletePodcastId !== ""){
+      shareStore.deleteItem(item, getDeletePodcastId, props.podcastId)
+    }
+    if(destination == props.podcastId || getDeletePodcastId ==props.podcastId){
```

Comment:
```suggestion
    if(destination == props.podcastId || getDeletePodcastId == props.podcastId){
```
eslint should have picked this up

Reply:
I have already removed it.